src/tribler-gui/tribler_gui/event_request_manager.py

```diff
@@ -71,7 +73,6 @@ def events_start_received(self, event_dict):
     def on_error(self, error, reschedule_on_err):
         self._logger.info(f"Got Tribler core error: {error}")
 
-        SentryReporter.ignore_logger(self._logger.name)
```

Comment:
**Issue:** without this line we will have unuseful garbage in the logs, see https://github.com/Tribler/tribler/pull/5821

Reply:
Addressed by logging only every 10th connection attempt.